src/sass/abstracts/_mixins.scss

```diff
@@ -0,0 +1,69 @@
+// MEDIA QUERY MANAGER
+
+/* 
+0 - 600px:       Phone
+600 - 900px:     Tablet portrait
+900 - 1200px:    Tablet landscape
+[1200 - 1800]:   Normal styles apply  
+1800px +:        Big desktop
+*/
+
+/* 
+$breakpoint arguement choices: 
+- phone
+- tab-port
+- tab-land
+- big-desktop
+*/
+
+// Media querys are not affected by html font-size
+// rems don't work well
+// 1em = 16px
+@mixin respond($breakpoint) {
+  @if $breakpoint == phone {
+    @media only screen and (max-width: 37.5em) {
+      //600px
+      @content;
+    }
+  }
+  @if $breakpoint == tab-port {
+    @media only screen and (max-width: 56.25em) {
+      //900px
+      @content;
+    }
+  }
+  @if $breakpoint == tab-land {
+    //1200px
+    @media only screen and (max-width: 75em) {
+      @content;
+    }
+  }
+  @if $breakpoint == big-desktop {
+    //1800px
+    @media only screen and (min-width: 112.5em) {
```

Comment:
Maybe this width should be an absolute unit? What do you think? Otherwise, it will change with the font size.
Same question with the other breakpoints.

Reply:
That's the idea! 
With this kind of media queries we can have a bigger font-size and dynamic measures in bigger resolutions (and it works the same way with smaller ones). 
This way general style adapts better to every screen.